runtime/doc/news.txt

```diff
@@ -189,6 +189,8 @@ The following new APIs and features were added.
 
 • Added |:fclose| command.
 
+• |complete| option supports `f` flag for completing buffer names.
```

Comment:
- We want to link to the 'complete' option docs here. Options are linked with single quotes in help docs. Also, there is no tag for `|complete|`.
- Flags should use double quotes.

```suggestion
• 'complete' option supports "f" flag for completing buffer names.
```

Reply:
make sense . learned :+1: